setup.py

```diff
@@ -25,22 +25,22 @@
 # therein for more information.
 sys.path.insert(0, os.path.dirname(__file__))
 
-try:
-    import versioneer
-    version_config = versioneer.get_version()
-    cmdclass = versioneer.get_cmdclass()
-except ImportError:
-    print("WARNING: failed to import versioneer, falling back to no version for now")
-    version_config = "0.1.0"  # Fallback version
-    cmdclass = {}
+# try:
+#     import versioneer
+#     version_config = versioneer.get_version()
+#     cmdclass = versioneer.get_cmdclass()
+# except ImportError:
+#     print("WARNING: failed to import versioneer, falling back to no version for now")
+#     version_config = "0.4.0"  # Fallback version
+#     cmdclass = {}
```

Comment:
Similar to https://github.com/lincbrain/linc-cli/pull/2/files#r1406776675 -- versioneer needs to be refined